artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/QueueControlImpl.java

```diff
@@ -618,6 +618,52 @@ public boolean isPurgeOnNoConsumers() {
       }
    }
 
+   @Override
+   public void disable() throws Exception {
+      if (AuditLogger.isEnabled()) {
+         AuditLogger.disable(queue);
+      }
+      checkStarted();
+
+      clearIO();
+      try {
+         server.getPostOffice().updateQueue(queue.getQueueConfiguration().setEnabled(false));
+      } finally {
+         blockOnIO();
+      }
+   }
+
+   @Override
+   public void enable() throws Exception {
+      if (AuditLogger.isEnabled()) {
+         AuditLogger.enable(queue);
+      }
+      checkStarted();
+
+      clearIO();
+      try {
+         server.getPostOffice().updateQueue(queue.getQueueConfiguration().setEnabled(true));
```

Comment:
now it updates the config, as such persists in the queue binding.

Reply:
nice one... I will run tests and merge it